apps/admin-portal/src/components/claims/wizard/external-dialect/external-claims-add-dialect.tsx

```diff
@@ -107,8 +107,9 @@ export const ExternalClaims = (props: ExternalClaimsPropsInterface): ReactElemen
                             )
                             : (
                                 <EmptyPlaceholder
-                                    title="No External Claim"
-                                    subtitle={ [ "Currently, there is no external claim available for this dialect." ] }
+                                    title="No External Attribute"
+                                    subtitle={ [ "Currently, there is no external attribute available for " + 
+                                    "this dialect." ] }
```

Comment:
This has to be Attributes right ?

Reply:
Ah yes it's referring to multiple items. So should be `Attributes`.

Reply:
Fixed with commit aeae82d